PhpStormStubsMap.php

```diff
@@ -11302,47 +11328,47 @@ final class PhpStormStubsMap
   'SE_NOPREFETCH' => 'imap/imap.php',
   'SE_UID' => 'imap/imap.php',
   'SID' => 'standard/standard_defines.php',
-  'SIGABRT' => 'pcntl/pcntl.php',
-  'SIGALRM' => 'pcntl/pcntl.php',
+  'SIGABRT' => 'swoole/constants.php',
```

Comment:
Please update the rebase your changes on a master branch & re-generate stubs map.
These changes are outdated because of https://github.com/JetBrains/phpstorm-stubs/pull/1353